docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -5,24 +5,54 @@ description: Managing your review workflows and stages in Strapi.
 
 # Managing Review Workflows <EnterpriseBadge />
 
-The Review Workflows feature allows you to create and manage any desired review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content) and [Content-Types Builder](/user-docs/content-type-builder), review workflows should be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
+The Review Workflows feature allows you to create and manage different workflows for different content-types. Each workflow can consist of any review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication.
+
+<ThemedImage
+  alt="Workflow list view"
+  sources={{
+    light: '/img/assets/review-workflows/list-view-light.png',
+    dark: '/img/assets/review-workflows/list-view-dark.png',
+  }}
+/>
+
+In many organizations different teams review different parts of content. By using different review workflows for different content-types, it is possible to adjust each workflows to the needs of each team involved. The default workflow is configured to have 4 stages: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
```

Comment:
```suggestion
In many organizations different teams review different parts of content. By using different review workflows for different content-types, it is possible to adjust each workflow to the needs of each team involved.

The default workflow is configured to have 4 stages: To do, In progress, Ready to review, and Reviewed. All 4 stages can be edited, reordered or deleted as needed, and it is also possible to add new stages.
```
Tiny typo + suggestion to split into 2 paragraphs to make it even more easy to read.